package.json

```diff
@@ -18,7 +18,7 @@
     "unzipper": "^0.10.11"
   },
   "devDependencies": {
-    "@types/node": "^12",
+    "@types/node": "^18",
```

Comment:
We don't actually depend on any Node 18 exclusive things, but the Node 16 types don't have the AbortSignal being an event emitter, which we need for `signal.addEventListener('abort'`

Reply:
I updated the readme https://github.com/microsoft/vscode-test#vscode-test